test/testautomation_stdlib.c

```diff
@@ -36,6 +35,16 @@ stdlib_strlcpy(void *arg)
   return TEST_COMPLETED;
 }
 
+#if defined(HAVE_WFORMAT) || defined(HAVE_WFORMAT_EXTRA_ARGS)
```

Comment:
I chose to "surgically" ignore these format warnings so format warnings in other locations are still reported.